draft-privacypass-rate-limit-tokens.md

```diff
@@ -106,15 +109,16 @@ protocol ({{ISSUANCE}}) can be used to verify that a client meets a particular
 bar for attestation, but cannot be used by a redeeming server to rate-limit
 specific clients.
 
-There are several common use cases for rate-limiting anonymous clients that
+There are several use cases for rate-limiting anonymous clients that
 are common on the Internet. These routinely use client IP address tracking,
 among other characteristics, to implement rate-limiting.
 
 One example of this use case is rate-limiting website accesses to a client to
 help prevent fraud. Operations that are sensitive to fraud, such as account
 creation on a website or logging into an account, often employ rate-limiting
 as a defense-in-depth strategy. Additional verification can be required by these
-pages when a client exceeds a set rate-limit.
+pages when a client exceeds a set rate-limit. See {{ANTIFRAUDUSECASES}} and related
+documents of the working group for an in depth discussion on this topic.
```

Comment:
It's not clear to me that those use cases require rate limiting as opposed to something like basic issuance, so I might omit this reference (for now).

Reply:
Done